manim/mobject/number_line.py

```diff
@@ -105,7 +107,7 @@ def get_tick_numbers(self):
     def number_to_point(self, number):
         alpha = float(number - self.x_min) / (self.x_max - self.x_min)
         return interpolate(
-            self.get_start(), self.get_end(), alpha
+            self.get_start()+self.add_start*RIGHT, self.get_end()-self.add_end*RIGHT, alpha
```

Comment:
PEP8; making it clearer
```suggestion
            self.get_start() + self.add_start * RIGHT,
            self.get_end() - self.add_end * RIGHT,
            alpha
```